### DISCUSSION TITLE
Config port

### DISCUSSION THREAD

**Discussion by fdeleon** [owncast/owncast#522] [Q&A] [answered]
Hello everybody! Owncast is an amazing solution!, so, I'm testing it on a shared hosting (none of the listed on documentation), but, port 8080 is already in use, I was looking in documentation if I can configure and change it, but I didn't find how to, could anybody tell me if it is possible?

best regards

**Comment by jeyemwey** [comment, marked as answer]
Hi, to change the web server port, add this to the root of your config.yaml:

```yaml
webServerPort: 8081
```

If you also want to change the RTMP port, youll have to rebuild owncast from source or wait for the 0.0.4 release. Then, you can change the port via: 


```yaml
rtmpServerPort: 1936
```

**Reply by fdeleon** [reply]
Thank you very much for your response, it worked fine, so, currently I only need to change webserver port, but, it will be so nice to change rtmp too. 

best regards

**Reply by geekgonecrazy** [reply]
Looks like someone added webServerPort to documentation.  I’ve opened https://github.com/owncast/owncast.github.io/pull/28 to add the one for rtmp
